compiler/ras/Debug.cpp

```diff
@@ -596,6 +596,12 @@ TR_Debug::vtrace(const char * format, va_list args)
    if (_file != NULL)
       {
       char buffer[256];
+      if(TR::Options::_traceFileLength && TR::IO::ftell(_file) > (TR::Options::_traceFileLength * (1 << 20)))
```

Comment:
This won't behave properly if `_traceFileLength >= 0x800`; the type is `int32_t` which will overflow.
I suggest the test should be:
```
      if ((0 != TR::Options::_traceFileLength) && TR::IO::ftell(_file) > (TR::Options::_traceFileLength * (1L << 20)))
```
(because the return type of `ftell()` is `long`).